app/index.ejs

```diff
@@ -3,21 +3,23 @@
 <head>
     <base href="/">
 
-    <title>Angular 2 QuickStart</title>
+    <title>Exadel: English Tests</title>
+
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1" user-scalable="no">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0 user-scalable="no" />
 
     <!--Import material icons-->
-    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link type rel="stylesheet" href="app/assets/fonts/materialize-include.css" >
     <link href=”https://maxcdn.bootstrapcdn.com/bootstrap/x.x.x/css/bootstrap.min.css” rel=”stylesheet”>
```

Comment:
Thats bad import, need investigate the rison of using bootstrap and get rid of it.